gvm/protocols/base.py

```diff
@@ -35,6 +39,8 @@ class GvmProtocol:
             See :mod:`gvm.transforms` for existing transforms.
     """
 
+    _UNDEFINED_VALUE = UndefinedValue()
```

Comment:
Personally I would change this into a module variable instead of a class variable but this is more or less just a matter of taste.